controllers/spacerequest/spacerequest_controller_test.go

```diff
@@ -392,6 +392,42 @@ func TestCreateSpaceRequest(t *testing.T) {
 		})
 	})
 
+	appstudioTier.Spec.SpaceRequestConfig.ServiceAccountName = "manager"
+	t.Run("failure service account not present", func(t *testing.T) {
```

Comment:
this is testing the case in which the service account doesn't exist. For this I would rather create a new tier that has a service account which doesn't exist or just configure gock at line 403 to return a different SA name, something like:
```
commontest.SetupGockForServiceAccounts(t, member1.APIEndpoint, types.NamespacedName{
				Name:      "another-sa-name",
				Namespace: "jane-env",
			})
```

Also, unless I'm missing something, I don't see a test that:
- uses a tier which has `.Spec.SpaceRequestConfig` unset
- verifies that spacerequest is provisioned with namespace access and no secretRef
- the subspace is ready with the given tiername from the spacerequest